.drone.yml

```diff
@@ -2,6 +2,10 @@ workspace:
   base: /go
 
 pipeline:
+  clone:
+    image: plugins/git
+    tags: true
```

Comment:
I'm not sure, if I use `tags` as `true` this is what I get:

```
Successfully built 5ee8ab55d52a
+ docker tag image quay.io/ipedrazas/drone-helm:latest
+ docker tag image quay.io/ipedrazas/drone-helm:${DRONE_TAG}
Error parsing reference: "quay.io/ipedrazas/drone-helm:" is not a valid repository/tag
```
as you can see `${DRONE_TAG}` is empty and yes, the clone tags block is added:

```
+ git init
Initialized empty Git repository in /go/src/github.com/ipedrazas/drone-helm/.git/
+ git remote add origin https://github.com/ipedrazas/drone-helm.git
+ git fetch --tags origin +refs/heads/clonetags:
From https://github.com/ipedrazas/drone-helm
 * branch            clonetags  -> FETCH_HEAD
 * [new branch]      clonetags  -> origin/clonetags
+ git reset --hard -q 5b91a1b48905b43d05311762635899d8a4b83043
+ git submodule update --init --recursive
```